### ISSUE TITLE
validation set label

### ISSUE THREAD

**Issue by TangKun1234** [Cadene/pretrained-models.pytorch#61]
I already got pretrained AlexNet model, and want to reproduce results, but i couldn't get the label of validation set.

**Comment by Cadene** [comment]
Hi @TangKun1234,

Please find the validation images here: `wget http://data.lip6.fr/cadene/imagenet/val.tar.gz`

I am working on a better solution.
